acceptance/options/embedded_db.rb

```diff
@@ -3,6 +3,18 @@
 #  a variable 'options_file_path', though.
 require File.expand_path(File.join(File.dirname(options_file_path), 'common.rb'))
 
-common_options_hash.merge({
-  :puppetdb_database => 'embedded',
-})
+defaults_opts = if options[:type] == 'aio' then
```

Comment:
I'm sorry @johnduarte I've given you bad advice here. The options file stuff is parsed by beaker, not by us. In fact its done in an order that means you can prefetch the options hash this way, you have to pluck the value from the environment instead - like ENV["BEAKER_TYPE"]. This sounds scarier than it sounds, the name of the variable is still a static API so its still not going to just underneath us (unless we do a major revision bump to beaker I guess, but it would break other stuff _anyway_ since our CI jobs would need changing).

I'll fire up a job tomorrow with this change before you get online and we can see if it solves it.